src/main/java/domain/board/Direction.java

```diff
@@ -0,0 +1,62 @@
+package domain.board;
+
+import java.util.Arrays;
+
+public enum Direction {
+
+    UP(0, 1),
+    UPRIGHT(1, 1),
+    RIGHT(1, 0),
+    DOWNRIGHT(1, -1),
```

Comment:
```suggestion
    DOWN_RIGHT(1, -1),
```
단어들 사이에 공백을 넣어주는건 어떨까요?